src/App.vue

```diff
@@ -67,6 +69,9 @@ export default {
           const languageResult = `${first}-${secondUpperCase}`;
           this.$i18n.locale = languageResult;
         }
+        const hlp = initHelpHero('m7dO0to4OK');
```

Comment:
I think it would be better to use this string as an env var, for security reasons